waspc/src/Parser/Route.hs

```diff
@@ -0,0 +1,26 @@
+module Parser.Route
+    ( route
+    ) where
+
+import Text.Parsec.String (Parser)
+
+import qualified Lexer as L
+import qualified Wasp.Route as Route
```

Comment:
I always wonder in situations like this if I should qualify it as Route, or just R, or maybe import type directly as Route and then qualify the rest as Route, or R. So many choices! It is somewhat ugly to see Route.Route in the code. On the other hand at least it is clear what it is. Hmmmm. Anyway this is ok I am just sharing thoughts.

Reply:
yes me too! I believe we will find something we like, but still trying out things for now.